### PR TITLE
#254 inline resume

### PR DESCRIPTION
**Pull request by bohraji**

### CODE REVIEW

File: resources/views/hr/application/inline-resume.blade.php [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<a href="#" data-trigger="resume"><i class="fa fa-file fa-2x"></i></a>
```
Comment (rathorevaibhav) [inline resources/views/hr/application/inline-resume.blade.php]: Not completely sure, but I don't think to have an anchor tag with href=# fits well here.
If we only need to show/hide the iFrame, I think a span/div would do the work. And that could be the reason why Bootstrap uses div for modal toggles.

**Comment by bohraji** [comment]
@rathorevaibhav i have updated the feedbacks. Please have a look.
Thanks for the help 😄

File: resources/views/hr/application/inline-resume.blade.php [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<span class="c-pointer text-primary" @click="toggleResume()"><i class="fa fa-file fa-2x"></i></span>
+<div class="card border-dark inline-resume" id="sliding_resume" v-bind:class="{ hidden: isHidden }">
```
Comment (rathorevaibhav) [inline resources/views/hr/application/inline-resume.blade.php]: `v-bind:class="{ hidden: isHidden }` can be replaced by `v-show="!isHidden"`

File: resources/views/hr/application/inline-resume.blade.php [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    <div class="card-header bg-warning">
+        <ul class="nav justify-content-end">
+            <li class="nav-item">
+                <b><a class="nav-link py-0" href="{{ $resume }}" target="_blank"><i class="fa fa-external-link" aria-hidden="true"></i> Open in a New Tab</a></b>
+            </li>
+            <li class="nav-item">
+                <b><span class="c-pointer text-primary" href="#" @click="toggleResume()">Close</span></b>
```
Comment (rathorevaibhav) [inline resources/views/hr/application/inline-resume.blade.php]: remove href

File: resources/assets/js/app.js [hunk 4]
```diff
@@ -28,11 +28,15 @@ if (document.getElementById('page_hr_applicant_edit')) {
     const applicantEdit = new Vue({
         el: '#page_hr_applicant_edit',
         data: {
+            isHidden: true,
```
Comment (rathorevaibhav) [inline resources/assets/js/app.js]: can we rename this? This element `page_hr_applicant_edit` contains multiple data and elements. It'll be better if we make this variable name and methods more specific to their use. something like `hideResumeFrame` and `toggleResumeFrame`

**Pull request merged**